terraform/cp4i/variables.tf

```diff
@@ -56,12 +56,21 @@ variable "entitled_registry_key" {
 variable "entitled_registry_user_email" {
   description = "Email address of the user owner of the Entitled Registry Key"
 }
+variable "on_vpc" {
+  default = "false"
+  description = "Required: Cluster type to be installed on, 'true = VPC, 'false' = Classic"
+}
+
+variable "flavors" {
+  type        = list(string)
+  default     = ["bx2.16x64"]
```

Comment:
Set the default to the classic flavor of ["b3c.16x64"].  We want all the defaults to provision on classic infrastructure.  The user will have to explicitly set on_vcp=true to use VPC.